.github/workflows/aws_tfhe_tests.yml

```diff
@@ -169,7 +169,7 @@ jobs:
       (github.event_name == 'pull_request' && needs.setup-instance.result != 'skipped')
     needs: [ should-run, setup-instance ]
     concurrency:
-      group: ${{ github.workflow }}_${{github.event_name}}_${{ github.head_ref || github.ref }}
+      group: ${{ github.workflow }}_${{github.event_name}}_${{ github.ref }}
```

Comment:
do you know why we have the event name here ?

Reply:
Oh i get it now, we have a `schedule` event occurring everyday @ 1a.m everyday.
For devs outside of Paris timezone, it might be troublesome if they have their CI in their PR being cancelled because of this `schedule` event.
I'll let the `event_name` but I'll refactor the couple `workflow` + `ref` using `workflow_ref` like elsewhere.